types/index.d.ts

```diff
@@ -1,32 +1,45 @@
-import { Application, Request } from 'express'
-import { FastifyPluginCallback, FastifyRequest } from 'fastify'
+import { Application, Request } from "express";
+import { FastifyPluginCallback, FastifyRequest } from "fastify";
 
-declare module 'fastify' {
+declare module "fastify" {
   interface FastifyInstance {
     /**
      * Express middleware function
      */
-    use: Application['use']
+    use: Application["use"];
 
     /**
+     * @private
      * Express application instance
      */
-    express: Application
+    express: Application;
+  }
+
+  interface FastifyContextConfig {
+    /**
+     * @private
```

Comment:
expressing that this type is a private type and one we probably shouldnt use in application code without good reason